crates/ark/src/lsp/symbols.rs

```diff
@@ -120,13 +120,56 @@ fn is_indexable(node: &Node) -> bool {
     true
 }
 
+// Function to parse a comment and return the section level and title
+fn parse_comment_as_section(comment: &str) -> Option<(usize, String)> {
+    // Match lines starting with one or more '#' followed by some non-empty content and must end with 4 or more '-' or '#'
```

Comment:
```suggestion
    // Match lines starting with one or more '#' followed by some non-empty content and must end with 4 or more '-', '#', or `=`
```

Reply:
That's right! Thank you for pointing this out. `=` is supported, in line with RStudio's behavior.